src/main/java/net/minecraftforge/gradle/GradleForgeHacks.java

```diff
@@ -0,0 +1,286 @@
+/*
+ * LegacyDev
+ * Copyright (c) 2016-2020.
```

Comment:
Correct me if I am wrong but I don't think this or any of the added files have the second line of the license match the change to the LICENSE-header.txt

Reply:
If we're changing the header, we should just move to the simplified header like we have for the rest of the projects.